server/src/core/user/user-service.ts

```diff
@@ -683,6 +684,103 @@ export class Users implements UserService {
     }
   }
 
+  public async addBeneficiary(args: UserAddBeneficiaryArgs): Promise<User> {
+    
+    validators.validatesAddBeneficiary(args);
+    const { phone, name, email } = args;
+    try {
+      const now = new Date();
+      const password = await hashPassword(generatePassword());
+      const user: DbUser = {
+        _id: generateId(),
+        phone,
+        name,
+        password,
+        addedBy: '',
+        isVetted: false,
+        donors: [],
+        roles: ['beneficiary'],
+        createdAt: now,
+        updatedAt: now
+      };
+
```

Comment:
This method should add be `addVettedBeneficiary`, the beneficiary should be `vetted` but not `verified`. Maybe instead of `vetted` we should find a less confusing word. But `vetted` here means that we as SocialRelief added the beneficiary ourselves (or through trusted partners), rather than the beneficiaries being added by donors on the platform.